apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -76,7 +98,7 @@ class MyMediaPlayer : public MediaPlayerObserverInterface,
 bool MyMediaPlayer::init(int test)
 {
 	if (mp.create() != PLAYER_OK) {
-		cout << "Mediaplayer::create failed" << endl;
+		meddbg( "Mediaplayer::create failed" "\n");
```

Comment:
`meddbg` is media framework level debug log.
I think just printf is enough

Reply:
I wasn't about to change cout logs (I changed it for debugging).
I have reverted it now.